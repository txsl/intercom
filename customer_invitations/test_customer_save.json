{"longitude": -19.123989, "user_id": 1, "name": "Tim Berners-Lee", "latitude": 45.231932}
{"longitude": -12.127389, "user_id": 2, "name": "Woz", "latitude": 23.412435}
{"longitude": -0.12389, "user_id": 3, "name": "Vint Cerf", "latitude": 55.123546}
