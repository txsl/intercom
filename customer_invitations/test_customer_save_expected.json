{"user_id": 1, "longitude": -19.123989, "name": "Tim Berners-Lee", "latitude": 45.231932}
{"user_id": 2, "longitude": -12.127389, "name": "Woz", "latitude": 23.412435}
{"user_id": 3, "longitude": -0.12389, "name": "Vint Cerf", "latitude": 55.123546}
